composer.lock

```diff
@@ -9,24 +9,24 @@
     "packages-dev": [
         {
             "name": "bamarni/composer-bin-plugin",
-            "version": "v1.3.0",
+            "version": "v1.2.0",
```

Comment:
I will do a "backport" for this today - the version bump in `master` should also be done in `stable10` and will eliminate this diff.